forum/forum/imprint.jsp

```diff
@@ -0,0 +1,27 @@
+<!--  INF16B Maximilian Gerlach -->
+<div class="imprint">
+    
+    <h2>Impressum</h2>
+    <div class="team">
+        <h3>Das Team</h3>
+        Die Studierenden des Kurses INF16B
```

Comment:
Die ganzen Textzeilen in dieser Datei sollten in einem `p` sein, damit man sie besser stylen kann und damit man kein `br` tag braucht.